### PR TITLE
Update Connext Adapter

### PR DESCRIPTION
**Pull request by ArjunBhuptani**
Updating Connext adapter to:
- Add new chains supported by the protocol
- Track xERC20s bridged by Connext to other chains

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/connext exports TVL: 
        
 
 
    ethereum                  782.67 M
    arbitrum                  5.36 M
    linea                     4.20 M
    optimism                  3.41 M
    mode                      2.44 M
    polygon                   2.31 M
    xdai                      2.18 M
    bsc                       1.44 M
    metis                     766.21 k
    base                      409.89 k
    
    total                    805.18 M

File: projects/connext/index.js [hunk 1]
```diff
@@ -49,21 +62,8 @@ const chains = [
   "optimism",
   "arbitrum",
   "mode",
-
```
Comment (g1nt0ki) [inline projects/connext/index.js:52]: can you re-add the deleted chains, else our server will break :(

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
code is mostly good, can you add back the chains, and exclude NEXT token from tvl? it is projects own token, we exclude it for other bridge/cross chain protocols.

**Comment by ArjunBhuptani** [comment]
Done! Thanks for the fast feedback :smile: @g1nt0ki

**Comment by llamatester** [comment]
The adapter at projects/connext exports TVL: 
        
 
 
    ethereum                  755.72 M
    arbitrum                  5.44 M
    linea                     4.11 M
    optimism                  3.36 M
    polygon                   2.30 M
    mode                      2.20 M
    xdai                      2.16 M
    bsc                       1.44 M
    metis                     748.81 k
    base                      378.24 k
    moonriver                 0
    fantom                    0
    moonbeam                  0
    fuse                      0
    cronos                    0
    milkomeda                 0
    boba                      0
    evmos                     0
    harmony                   0
    avax                      0
    
    total                    777.85 M

**Comment by g1nt0ki** [comment]
@ArjunBhuptani thanks for the PR

**Pull request merged**